src/components/StepDiff.tsx

```diff
@@ -97,30 +63,6 @@ export default class StepDiff extends React.PureComponent<
     return endRenderContent;
   };
 
-  renderIcon = (type: string): React.ReactNode => {
-    const mapTypeToSvg: {
-      [index: string]: string;
-    } = {
-      up,
-      down,
-    };
-
-    return <Image src={mapTypeToSvg[type]} />;
-  };
-
-  toggleCollapse = (fileName: string): void => {
-    console.log(fileName);
-    const { collapseObj } = this.state;
-    const newCollapseObj = {
-      ...collapseObj,
-      [fileName]: !collapseObj[fileName],
-    };
-
-    this.setState({
-      collapseObj: newCollapseObj,
-    });
-  };
-
   getCollapseObj = (arr: ChangedFile[]): CollapseObj => {
```

Comment:
This `getCollapseObj` is also redundant.

Reply:
We need maintain an array of need hide files, uh !